MetaMask/metamask-extension
# Replace setInterval with chrome alarms for MetaMetrics FinalizeEventFragment

## Explanation
We are replacing setInterval with chrome alarm to schedule the call to `FinalizeEventFragment()` every 1 min.  The current interval window for FinalizeEventFragment() is 30 sec; we are updating that to 1 min since chrome alarms require the interval to be at least 1 mins and replace setInterval with chrome alarms.



## More Information


Fixes #15933 
## Screenshots/Screencaps



### Before



### After



## Manual Testing Steps
1. Load the application, and in the service worker, try using breakpoints to check if the alarm is getting created.
2. Breakpoint or console log inside `chrome.alarm.onalarm.addListener()` and inside `getall()` to  verify the alarm is getting triggered at 1 mins interval. 


## Pre-Merge Checklist


### + If there are functional changes:
